ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -229,15 +229,23 @@ tests:
   steps:
     workflow: stackrox-stackrox-e2e-job
 - always_run: false
-  as: powervs-qa-e2e-tests
+  as: powervs-qa-ebpf-e2e-tests
   optional: true
   steps:
     env:
       COLLECTION_METHOD: ebpf
```

Comment:
You will need `TEST_SUITE: powervs-qa-e2e-tests` in all the jobs due to the name change. In order to find the job in stackrox scripts/ci/jobs.

Reply:
Yes, makes sense. I missed that. I will add it.